draft-packages/form/docs/CheckboxGroup.stories.tsx

```diff
@@ -19,42 +19,20 @@ interface Props {
   render: (props: RenderProps) => JSX.Element
 }
 
-class CheckboxGroupExample extends React.Component<Props> {
-  public state = {
-    checkedStatus: "mixed",
+function CheckboxGroupExample({ render }) {
```

Comment:
```suggestion
const CheckboxGroupExample: React.VFC<Props> = ({ render }) => {
```

I would probably change `Props` to `CheckboxGroupExampleProps` too, but up to you

Reply:
Thanks, I'll have a look. As it's a story I wasn't too concerned with "TS'ing" it, as I probably should have been.